gems/sorbet/bin/srb

```diff
@@ -33,11 +33,7 @@ shift
 srb_path="${BASH_SOURCE[0]}"
 
 compute_md5() {
-  if command -v md5sum > /dev/null; then
-    md5sum "$1"
-  else
-    md5 -r "$1"
-  fi
+  ruby -e "require 'digest'; puts Digest::MD5.hexdigest(File.read(ARGV[0]))" "$1"
```

Comment:
this is going to get slower by a bunch and i'm not in love with that since we pay the cost on EVERY invocation. Is there some other way? Can we depend on md5 and package it or something? Shelling out to ruby will basically double our `srb tc` cost

Reply:
If it's not important which hash it is, `shasum -a 256` works just fine.

Reply:
It is totally not important. Is `shasum` more present on more systems? If so, go with that! You'll have to update one location in ruby too, but I hope it has sha256 bindings.

Reply:
Hah, which is also a perl script :|

Reply:
It's looking like I'll be adding support for building md5 command line utilities in this PR instead.

Reply:
What about this third option: we fall back on using ruby's md5 implementation if both `md5sum` and `md5` are missing?